src/Pim/Bundle/CatalogBundle/EventSubscriber/ComputeProductRawValuesSubscriber.php

```diff
@@ -58,17 +59,21 @@ public static function getSubscribedEvents()
     public function computeRawValues(GenericEvent $event)
     {
         $product = $event->getSubject();
-        if (!$product instanceof ProductInterface) {
+        if (!$product instanceof ProductInterface || !$product instanceof ProductTemplateInterface) {
```

Comment:
This is kind of dirty, I think it will need its own subscriber. It is a quick win to move on quickly.